test-bench-utils/lib/sinks/xssRenderedTemplates.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const Pug = require('pug');
+const Ejs = require('ejs');
+const Handlebars = require('handlebars');
+
+/**
+ * @param {Object} params
+ * @param {string} params.input user input string
+ * @param {Object} opts
+ * @param {boolean=} opts.safe are we calling the sink safely?
+ * @param {boolean=} opts.noop are we calling the sink as a noop?
+ */
+module.exports.pug = async function pug(
+  { input },
+  { safe = false, noop = false } = {}
+) {
+  if (noop) return 'NOOP';
+
+  const template = safe ? `p='${input}'` : `p!='${input}'`;
+  return Pug.compile(template)();
+};
+
+/**
+ * @param {Object} params
+ * @param {string} params.input user input string
+ * @param {Object} opts
+ * @param {boolean=} opts.safe are we calling the sink safely?
+ * @param {boolean=} opts.noop are we calling the sink as a noop?
+ */
+module.exports.ejs = async function ejs(
+  { input },
+  { safe = false, noop = false } = {}
+) {
+  if (noop) return 'NOOP';
+
+  const template = safe ? `<p><%=input%></p>` : `<p><%-input%></p>`;
```

Comment:
If we're not using interpolation let's avoid using template strings:
```suggestion
  const template = safe ? '<p><%=input%></p>' : '<p><%-input%></p>';
```